packages/notifi-react-card/lib/components/subscription/subscription-card-views/HistoryCardView.tsx

```diff
@@ -148,7 +148,7 @@ export const AlertHistoryView: React.FC<AlertHistoryViewProps> = ({
           classNames?.dividerLine,
         )}
       />
-      {alertHistoryData?.nodes ? (
+      {alertHistoryData?.nodes && alertHistoryData?.nodes?.length > 0 ? (
```

Comment:
i'd prefer to see this pulled out so we don't have an implicit conversion

e.g.: on line 128, add `const nodeLength = alertHIstoryData?.nodes?.length ?? 0;` and here, use `nodeLength > 0`

Reply:
Thank you for letting me know about implicit conversions.

I learned that implicit conversion happens when expression contains operands of different built-in types, and no explicit casts are present, the compiler converts one of the operands so that the types match. 

The reason why `const nodeLength = alertHIstoryData?.nodes?.length ?? 0`; will not run into implicit conversion issues is because it will return 0 if `alertHistoryData?.nodes?.length` is `null `or `undefined `or `alertHistoryData?.nodes?.length value` if it is not null or undefined. 

In other words, it will always return a number.